{"statusDetails": {"trace": "self = <scripts.test_add_user.Test_Add_User object at 0x0000027034F56710>, name = 'lousifa2', phone = '22334546566'\n\n    @pytest.mark.parametrize(\"name,phone\",[(\"lousifa1\",\"13145111\"),(\"lousifa2\",\"22334546566\")])\n    def test_add_user(self,name,phone):\n>       self.obj_data.add_username_phone(name,phone)\n\nscripts\\test_add_user.py:25: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nPage\\add_user_page.py:19: in add_username_phone\n    self.input_element(Page.add_user_name,name)\nBase\\Base.py:23: in input_element\n    input = self.find_element(loc)\nBase\\Base.py:14: in find_element\n    return   WebDriverWait(self.driver,timeout,poll).until(lambda x: x.find_element(*loc))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"cf0b50cf-2095-401e-b59e-0409c0e9a5c5\")>, method = <function Base.find_element.<locals>.<lambda> at 0x0000027034F5B9D8>\nmessage = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value is not False.\"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\nd:\\python\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:80: TimeoutException", "message": "selenium.common.exceptions.TimeoutException: Message:"}, "stop": 1584936054041, "name": "test_add_user[lousifa2-22334546566]", "historyId": "001ea0fc15b1acb1e3e98058ec83c2c3", "start": 1584936043528, "fullName": "scripts.test_add_user.Test_Add_User#test_add_user", "parameters": [{"name": "name", "value": "'lousifa2'"}, {"name": "phone", "value": "'22334546566'"}], "uuid": "6509121e-b375-419d-a924-fde8217bba13", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "subSuite", "value": "Test_Add_User"}, {"name": "suite", "value": "test_add_user"}, {"name": "host", "value": "lou"}, {"name": "thread", "value": "7164-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_add_user"}], "status": "broken", "testCaseId": "b28d621880ac67dbee2712ada26e7f30"}